akkaStreamsCustomStage/src/main/java/com/lightbend/java/modelserver/modelserver/AkkaModelServer.java

```diff
@@ -72,6 +72,7 @@ public static void main(String [ ] args) throws Throwable {
         ReadableModelStore modelStateStore =
                 modelPredictions
                         .to(Sink.ignore())      // we do not read the results directly
+                        // try changing Sink.ignore() to Sink.foreach(x -> System.out.println(x))) What gets printed?
```

Comment:
Here nothing will should be printed - this just materialize state store for HTTP, or I am missing something?

Reply:
nm, tried it and verify. It will print the result returned by model predictions, above